OtripleS.Portal.Web.Tests.Unit/Views/StudentRegistrationComponents/StudentRegistrationComponentTests.Exceptions.cs

```diff
@@ -29,37 +30,37 @@ public partial class StudentRegistrationComponentTests
                     .ThrowsAsync(studentViewValidationException);
 
             // when
-            this.renderedStudentRegistrationComponent =
+            this.renderedStudentRegistrationComponents =
```

Comment:
@KFitz114 the component itself can still singular. but the namespace (the folder and the namespace text) needs to be plural.